drop-small-mss/drop-small-mss.yml

```diff
@@ -0,0 +1,39 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: drop-small-mss
+  namespace: kube-system
+  labels:
+    app: drop-small-mss
+spec:
+  selector:
+    matchLabels:
+      app: drop-small-mss
+  template:
+    metadata:
+      labels:
+        app: drop-small-mss
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ""
+    spec:
+      hostPID: true
```

Comment:
set hostnewtork to true?

Reply:
It's not necessary. We enter the host everything namespace when we execute the startup script.